e2e/testdata/fn-render/fn-failure-output-no-truncate/.expected/config.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# One of the functions in the pipeline fails resulting in
+# non-zero exit code and no changes in the resources.
```

Comment:
So `kubeval` fails here because schema isn't available for the custom resource ? (And that is because we invoke it in strict mode ?). We should update the comment.